server/service/src/sync/translations/invoice_line.rs

```diff
@@ -90,6 +90,8 @@ pub struct LegacyTransLineRow {
     #[serde(deserialize_with = "empty_str_as_option_string")]
     #[serde(rename = "linked_transact_id")]
     pub linked_invoice_id: Option<String>,
+    #[serde(deserialize_with = "empty_str_as_option_string")]
+    pub donor_default_id: Option<String>,
```

Comment:
It's called "donor_id" in OG, if we want to rename to include `default_`, you'll nee the serde rename, like the field above